middlewares/validators/tasks.js

```diff
@@ -133,8 +134,28 @@ const updateSelfTask = async (req, res, next) => {
   }
 };
 
+const getTasks = async (req, res, next) => {
+  const schema = joi.object().keys({
+    dev: joi.bool().optional().sensitive(),
+    status: joi
+      .string()
+      .insensitive()
+      .valid(...MAPPED_TASK_STATUS_ENUM)
+      .optional(),
+  });
+
```

Comment:
Nit: What i understand we validate `status: "UNASSIGNED"` and If I send `status: unassigned` then it validator reject or pass??

Reply:
The vaildator does a case insensitive check. That means even if it's `unassigned` or `UNASSIGNED`, both passes